File: 0162-find-peak-element/0162-find-peak-element.c
int findPeakElement(int* nums, int n) {
    if(n == 0) return -1;
    int max = INT_MIN,in=0;
    for(int i =0 ;i < n;i++){
        if(nums[i] > max) {max = nums[i]; in = i;}
    }
    return in;
}

File: 0162-find-peak-element/0162-find-peak-element.java
class Solution {
    public int findPeakElement(int[] nums) {
        int left = 0;
        int right = nums.length - 1;
        int mid = 0;
        while(left < right){
            mid = left+(right - left) / 2;
            if(nums[mid] > nums[mid + 1]){
                right = mid;
            }
            else{
                left = mid + 1;
            }
        }
        return left;
    }
}